vars/mavenExecute.groovy

```diff
@@ -66,6 +66,13 @@ def call(Map parameters = [:]) {
             command += " ${mavenFlags}"
         }
 
+        final disableDownloadsFlag = ' --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
```

Comment:
[minor] ```--batch-mode``` makes IMO sense anyway regardless how we set the log level for mvn downloads.

Reply:
Agreed, I updated the PR and resolved the merge conflicts.